.github/release.yml

```diff
@@ -0,0 +1,23 @@
+changelog:
+  exclude:
+    authors:
+      - micronaut-build
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - breaking
+    - title: New Features 🎉
+      labels:
+        - enhancement
+    - title: Bug Fixes 🐞
+      labels:
+        - bug
```

Comment:
I think the labs may be wrong here. Since you would need "type: bug"

Reply:
I see 🙈